k8s/namespaces/divorce/div-da/demo.yaml

```diff
@@ -6,7 +6,6 @@ spec:
   releaseName: div-da
   values:
     nodejs:
-      image: hmctspublic.azurecr.io/div/dn:pr-252-2fa3313-20220810085324 #{"$imagepolicy": "flux-system:demo-div-da"}
```

Comment:
you need this or it will use the base `image`

Reply:
fixed